### PR TITLE
[Docs] Update Japanese translation of feature_tap_dance.md.

### PR DESCRIPTION
**Pull request by shelaf**
## Description

Japanese documentation update.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

tag: @mtei

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_tap_dance.md [hunk 1]
```diff
@@ -359,189 +341,186 @@ qk_tap_dance_action_t tap_dance_actions[] = {
...
-void ql_reset (qk_tap_dance_state_t *state, void *user_data) {
-  //キーを押し続けていて今離したら、レイヤーをオフに切り替えます。
-  if (ql_tap_state.state==SINGLE_HOLD) {
-    layer_off(_MY_LAYER);
-  }
-  ql_tap_state.state = 0;
+void ql_reset(qk_tap_dance_state_t *state, void *user_data) {
+    // キーを押し続けていて今離したら、レイヤーをオフに切り替えます。
+    // If the key was held down and now is released then switch off the layer
+    if (ql_tap_state.state == SINGLE_HOLD) {
```
Comment (s-show) [inline docs/ja/feature_tap_dance.md:505]: 原文が訳文の中に残っています。
```suggestion
    // キーを押し続けていて今離したら、レイヤーをオフに切り替えます。
    if (ql_tap_state.state == SINGLE_HOLD) {
```

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by noroadsleft** [comment]
ありがとうございました！

**Pull request merged**